runtime/app.go

```diff
@@ -75,11 +75,6 @@ func (a *App) RegisterModules(modules ...module.AppModule) error {
 
 // Load finishes all initialization operations and loads the app.
 func (a *App) Load(loadLatest bool) error {
-	// register runtime module services
-	if err := a.registerRuntimeServices(); err != nil {
```

Comment:
Removing this breaks simapp. You need to rebase to main and add `runtime` in the `genesisModuleOrder` variable.